data.sql

```diff
@@ -45,4 +45,22 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander','Squirtle','Blossom');
-UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
+/*PROJECT DAY 4 */
+INSERT INTO vets (name, age, date_of_graduation) VALUES 
+  ('William Tatcher', 45, '2000-04-23'), 
+  ('Maisy Smith', 26, '2019-01-17'), 
+  ('Stephanie Mendez', 64, '1981-05-04'), 
+  ('Jack Harkness', 38, '2008-01-08');
+
```

Comment:
- [ ] Kindly update the date_of_graduation for Jack Harness because it is **2008-06-08** not **2008-01-08** in the requiements 👍